src/app/components/test-card-detailed/test-card-detailed.component.ts

```diff
@@ -36,13 +36,20 @@ export class TestCardDetailedComponent {
 
   movieStory: string
 
-  movieWriter:string
+  movieWriter: string
 
   imageTopActors: string[]
 
   // cast: IMovieCastPeople
   cast: any
 
+  movieExternalSourcesDetails: IMovieExternalSourcesDetails
+
+  movieExternalImdb_id: string
```

Comment:
i think it could be moved to an object named smth like `externalSources` to make this fields role more obvious

Reply:
movieExternalSourcesDetails removed